src/utils/OwnableWhitelist.sol

```diff
@@ -27,4 +27,20 @@ abstract contract OwnableWhitelist is Ownable {
     function toggleWhitelist(address client) external onlyOwner {
         whitelist[client] = !whitelist[client];
     }
+
+    /// @notice add a batch of addresses to the whitelist.
+    /// @param addrs an array of addresses to add.
+    function addToWhitelist(address[] calldata addrs) external onlyOwner {
+        for (uint256 i; i < addrs.length; i++) {
+            whitelist[addrs[i]] = true;
+        }
+    }
```

Comment:
Is there a way to iterate over all whitelisted operators ? Because right now it's map so it doesn't look like it.
But we may need that functionality, no ?

Reply:
even if only for offchain reason.

Reply:
We could run a query and prove it off chain ;)